extra/patterns/diamond.js

```diff
@@ -0,0 +1,29 @@
+//Run a loop row number of times for the pattern to be created.
+var arg = process.argv
+
+function Diamond(size){
+    if(size>=2 && size <=10){  // Condition to check if the input argument is within the range.
+        for (let i = 1; i <= size; i++) {
+            for (let j = 1; j <= size - i; j++) {
+                process.stdout.write(' ')
```

Comment:
use string concatination instead of process.out

Reply:
String concatenation is used instead of process.stdout method.
Changes implemented successfully.